src/python/pants/option/global_options.py

```diff
@@ -433,6 +433,14 @@ def register_bootstrap_options(cls, register):
             "Patterns use the gitignore syntax (https://git-scm.com/docs/gitignore). "
             "The `--pants-distdir` and `--pants-workdir` locations are inherently ignored.",
         )
+        register(
+            "--pants-gitignore",
```

Comment:
I recommend `--pants-use-gitignore` or even `--pants-use-gitignores`. `--pants-gitignore` made me think that this would be a list of file paths to the gitignores you want to use.